src/cljs/org/broadinstitute/firecloud_ui/common/table.cljs

```diff
@@ -230,12 +229,14 @@
       (map-indexed
         (fn [i column]
           (when (get column :reorderable? true)
-            [:div {}
-             (let [showing? (:showing? column)]
-               [:input {:type "checkbox"
+            (let [showing? (:showing? column)
+                  id (str "cb" i)]
+              [:div {}
+               [:input {:type "checkbox" :id id
                         :checked showing?
-                        :onChange #((:submit props) (update-in (:columns props) [i] assoc :showing? (not showing?)))}])
-             [:span {:style {:paddingLeft "0.5em"}} (:header column)]]))
+                        :onChange #((:submit props) (update-in (:columns props) [i] assoc :showing? (not showing?)))}]
+               [:label {:htmlFor id :style {:cursor "pointer"}}
```

Comment:
Prefer:
```clojure
 [:label {:style {:cursor "pointer"}} [:input {:type "checkbox" :checked showing?}] "label text here"]
```
rather than the IDs.